### PR TITLE
s2575, ta 13130: parse critwall config file

### PR DESCRIPTION
**Pull request by hjeeves**

### CODE REVIEW

File: critwall/src/main/java/org/opencadc/critwall/Main.java [hunk 1]
```diff
@@ -67,51 +67,198 @@
...
             Log4jInit.setLevel("org.opencadc.reg", cfg);
-            
-            // parse config file and populate/assign these
+            Log4jInit.setLevel("org.opencadc.critwall", cfg);
+
+            log.debug("log level: " + cfg);
+            log.debug("properties: " + props.toString());
+
+            // parse config file
+            String errMsg = "";
```
Comment (pdowler) [inline critwall/src/main/java/org/opencadc/critwall/Main.java]: Use a StringBuilder and append when there are more than a couple of strings ops to do.

File: critwall/src/main/java/org/opencadc/critwall/Main.java [hunk 2]
```diff
@@ -67,51 +67,198 @@
...
             System.exit(-1);
+
         }
+        log.debug("finished critwall run.");
     }
     
     private Main() { 
     }
+
+    static final String getSingleProperty(MultiValuedProperties props, String key) {
```
Comment (pdowler) [inline critwall/src/main/java/org/opencadc/critwall/Main.java]: getSingleProperty no longer needed - use MultiValuedProperties.getFirstPropertyValue directly

File: critwall/src/main/java/org/opencadc/critwall/Main.java (hunk 1 above)
Comment (hjeeves) [reply critwall/src/main/java/org/opencadc/critwall/Main.java]: ok

File: critwall/src/main/java/org/opencadc/critwall/Main.java (hunk 2 above)
Comment (hjeeves) [reply critwall/src/main/java/org/opencadc/critwall/Main.java]: done

**Pull request merged**